packages/react-intersection-observer/src/test/IntersectionObserver.test.tsx

```diff
@@ -27,42 +28,52 @@ describe('<IntersectionObserver />', () => {
 
   describe('wrapperComponent', () => {
     it('uses a div by default', () => {
-      const intersectionObserver = mount(<IntersectionObserver />);
-      const child = intersectionObserver.childAt(0);
-      expect(child.type()).toBe('div');
+      const intersectionObserver = mount(
+        <IntersectionObserver {...defaultProps} />,
+      );
+      const child = intersectionObserver.children[0];
+      expect(child).toHaveProperty('type', 'div');
     });
 
     it('uses a custom element', () => {
       const wrapperComponent = 'span';
       const intersectionObserver = mount(
-        <IntersectionObserver wrapperComponent={wrapperComponent} />,
+        <IntersectionObserver
+          {...defaultProps}
+          wrapperComponent={wrapperComponent}
+        />,
       );
-      const child = intersectionObserver.childAt(0);
-      expect(child.type()).toBe(wrapperComponent);
+      const child = intersectionObserver.children[0];
+      expect(child).toHaveProperty('type', wrapperComponent);
     });
 
     it('attaches the observer to the top-level node', () => {
       isSupported.mockReturnValue(true);
 
-      const intersectionObserver = mount(<IntersectionObserver />);
-      const child = intersectionObserver.childAt(0);
-      const node = child.getDOMNode();
+      const intersectionObserver = mount(
+        <IntersectionObserver {...defaultProps} />,
+      );
+      const child = intersectionObserver.children[0];
 
       expect(intersectionObserverMock.observers[0]).toHaveProperty(
         'target',
-        node,
+        child.domNode,
       );
     });
   });
 
   describe('children', () => {
     it('includes the children inside the top-level element', () => {
-      const children = <div>Hello world</div>;
+      const children = <div id="HelloWorld" />;
```

Comment:
I don't really get why we declare this outside of the jsx, it makes me expect that we will reference it down below but we don't. Maybe just have `HelloWorld` in a string we use in both places, but have the children inline?

Reply:
👍